### PR TITLE
add GSoC Intro Sem 2024 Autumn Slides

### PR DESCRIPTION
**Pull request by Majestic9169**
also updated index and provided canva link in readme

### CODE REVIEW

File: 2024/Autumn/GSoC-Intro-Seminar/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Google Summer of Code 2025 Intro Seminar
+
+**20th October 2024, 2:00pm, NR121**
+
+**Speakers:** 
+1. **Rohan Barsagade** @Wellcome Sanger Tree of Life, *GSoC'24*
+1. **Ankit Meda** @NumFOCUS, *GSoC'24*
```
Comment (Bikram-ghuku) [inline 2024/Autumn/GSoC-Intro-Seminar/README.md:7]:
```suggestion
2. **Ankit Meda** @NumFOCUS, *GSoC'24*
```

Comment (Dami-18) [reply 2024/Autumn/GSoC-Intro-Seminar/README.md:7]: LIF slides also need to be updated

Comment (Majestic9169) [reply 2024/Autumn/GSoC-Intro-Seminar/README.md:7]: image

markdown formats that correctly on its own

Comment (Majestic9169) [reply 2024/Autumn/GSoC-Intro-Seminar/README.md:7]: >  LIF slides also need to be updated

this is a separate issue not meant for this channel or this pr

**Pull request merged**